packages/core/src/constants.ts

```diff
@@ -176,10 +176,3 @@ export enum PostMessageMethods {
 }
 
 export const SUPPORTED_IMAGE_FORMATS = ['jpg', 'png', 'webp', 'gif', 'avif'] as const;
-export const SUPPORTED_IMAGE_OBJECT_FIT = [
```

Comment:
this is probably okay overall. The one gotcha I could see is if we wanted to support another option that was removed, that would require and SDK update. Did you work with travis on removing some of these?